src/pytorch_lightning/strategies/tpu_spawn.py

```diff
@@ -146,8 +149,8 @@ def distributed_sampler_kwargs(self) -> Dict[str, int]:
 
     @property
     def is_distributed(self) -> bool:
-        # HOST_WORLD_SIZE is None outside the xmp.spawn process
-        return os.getenv(xenv.HOST_WORLD_SIZE, None) and self.world_size != 1
+        # HOST_WORLD_SIZE is not set outside the xmp.spawn process
+        return (xenv.HOST_WORLD_SIZE in os.environ) and self.world_size != 1
```

Comment:
Reviewer: This was a bug. The statement `None and whatever `would return `None` here, instead of a bool. The intention was that we return a bool.